plugins/image-prioritizer/tests/test-helper.php

```diff
@@ -33,7 +33,7 @@ public function data_provider_to_test_image_prioritizer_init(): array {
 				'expected' => false,
 			),
 			'with_new_version' => array(
-				'version'  => '0.7.0',
+				'version'  => '99.0.0',
```

Comment:
why the giant version bump?

Reply:
I had done this before for Embed Optimizer in 2c799add22ee5cdda8f795b6aa555a56ae0c3b03 but neglected to do the same here. This test is to ensure that the init logic for Image Prioritizer does if the installed version of Optimization Detective required is greater than or equal to the required version. I didn't have much foresight when I picked the original `0.7.0` version since that version soon became the actual version. So this picks a version way off into the future (which will never be reached) to test that the logic always runs as expected.

Reply:
Ah, makes sense. reading out of context it was confusing.